experimental/packages/api-logs/src/types/Logger.ts

```diff
@@ -27,8 +26,10 @@ export interface Logger {
 
   /**
    * Emit an event. This method should only be used by instrumentations emitting events.
-   *
-   * @param event
+   *  
+   * @param eventName the event name, which acts as a classifier for events. Within a particular event domain,
+   * event name defines a particular class or type of event.
+   * @param logRecord
    */
-  emitEvent(event: LogEvent): void;
+  emitEvent(eventName: string, logRecord: LogRecord): void;
```

Comment:
This is mentioned here in the specification. `domain` should be defined by `Logger`. It is recommended to use different `loggers` to manage different `domian`, and the `eventName` should be determined when submitting an event. its data model is `LogRecrod` too. So the entries here should be `name` and `logrecord`.
![image](https://user-images.githubusercontent.com/12229677/208228381-1cd9d33e-afff-4287-96ec-60d5bc784df1.png)
source: https://opentelemetry.io/docs/reference/specification/logs/event-api/

Reply:
With this change, I find that the `domain` is missing from the API, which should be necessary as part of the event API.

We don't have an EventLogger interface right now, which should provide a static method to create an EventLogger instance with a `domain` name. I believe we should address this as a whole so that we won't miss anything from the API.

Reply:
Sorry, it seems like the `domain` is present in the `LoggerOptions`. It makes sense now.